From 0fd29435ce41d60cdb3b5e06e491500ddc54a86b Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Wed, 22 Mar 2017 17:39:37 +0200
Subject: [PATCH] selftests: seccomp: use LDLIBS to link against libpthread

Use the same fixup as used for timers test:
use LDLIBS to properly link against libpthread.

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>
---
 tools/testing/selftests/seccomp/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/seccomp/Makefile
+++ b/tools/testing/selftests/seccomp/Makefile
@@ -1,6 +1,6 @@
 TEST_PROGS := seccomp_bpf
 CFLAGS += -Wl,-no-as-needed -Wall
-LDFLAGS += -lpthread
+LDLIBS += -lpthread
 
 all: $(TEST_PROGS)
 
